src/core_modules/capture-core/components/Pages/New/NewPage.container.js

```diff
@@ -68,7 +68,7 @@ export const NewPage: ComponentType<{||}> = () => {
     // This is combo category selection. When you have selected a program but
     // the selection is incomplete we want the user to see a specific message
     const programCategorySelectionIncomplete: boolean =
-      useSelector(({ currentSelections: { programId, complete } }) => programId && !complete);
```

Comment:
Also figured out there was a bug here because I was looking into the `currentSelections` to verify wether or not programs and org units where present. However looking at the url is for this case a better solution